test/integration/check-ins/check-in.js

```diff
@@ -27,7 +27,7 @@ context('patient check-in', function() {
       .get('.check-in__frame')
       .find('.check-in__context-trail')
       .should('contain', 'Test Patient')
-      .should('contain', formatDate(completedDate, 'LONG'));
+      .should('contain', formatDate(moment.utc(completedDate), 'LONG'));
```

Comment:
I'm not sure this is right.. I think it needs to be `moment.utc(completedDate).local()`
```
moment.utc('2020-06-17 16:05:22').format('lll')
"Jun 17, 2020 4:05 PM"
moment.utc('2020-06-17 16:05:22').local().format('lll')
"Jun 18, 2020 1:05 AM"
moment('2020-06-17T16:05:22Z').format('lll')
"Jun 18, 2020 1:05 AM"
```